api/handlers/certificates.go

```diff
@@ -89,21 +90,13 @@ func (h AppHandler) CertCreate(c echo.Context) error {
 	}
 
 	// Validating JWT before any other action
-	token, err := ValidateJWT(c, h.config)
+	ca := auth.OpenIDCAuth{}
+	username, err := ca.Authenticate(c, h.config)
 	if err != nil {
 		return c.JSON(http.StatusUnauthorized,
 			map[string]string{"result": "fail", "message": "Failed validating JWT", "details": err.Error()})
```

Comment:
Maybe a proper message here would be `"Authentication failed"`, as `ca.Authenticate` is now being used?

Reply:
Done!